sources/packages/web/src/views/supporting-user/SupportingInformation.vue

```diff
@@ -158,13 +158,41 @@ export default defineComponent({
       // Disable internal submit button.
       formioUtils.disableWizardButtons(formInstance);
       formInstance.options.buttonSettings.showSubmit = false;
+      // Handle the navigation using the breadcrumbs.
+      formInstance.on(
+        "wizardPageSelected",
+        (page: WizardNavigationEvent, index: number) => {
+          isFirstPage.value = index === 0;
+          isLastPage.value = formInstance.isLastPage();
+          // Event to set isInFirstPage, current page and isInLastPage to parent
```

Comment:
for comment, we use sentence case ending with a full stop.  you can add a full stop here and line 180